lib/vue/src/components/input/CInputColorPicker.vue

```diff
@@ -1,53 +1,57 @@
 <template>
   <div>
-    <b-button
-      :style="`color: ${pickedColor}; fill: ${pickedColor};`"
-      class="p-0 rounded-circle bg-white border-white shadow-none"
-      @click="toggleMenu"
-    >
-      <svg
-        viewBox="0 0 32 32"
-        style="width: 32px; height: 32px;"
-        class="border border-light rounded-circle"
-      >
-        <pattern
-          id="checkerboard"
-          width="12"
-          height="12"
-          patternUnits="userSpaceOnUse"
-          fill="FFF"
+    <div class="d-flex align-items-center">
+      <b-button
+          :style="`color: ${pickedColor}; fill: ${pickedColor};`"
+          class="p-0 rounded-circle bg-white border-white shadow-none"
+          @click="toggleMenu"
         >
-          <rect
-            fill="#7080707f"
-            x="0"
-            width="6"
-            height="6"
-            y="0"
-          />
-          <rect
-            fill="#7080707f"
-            x="6"
-            width="6"
-            height="6"
-            y="6"
-          />
-        </pattern>
-
-        <circle
-          cx="16"
-          cy="16"
-          r="16"
-          fill="url(#checkerboard)"
-        />
-
-        <circle
-          cx="16"
-          cy="16"
-          r="16"
-        />
-      </svg>
-    </b-button>
-
+          <svg
+            viewBox="0 0 32 32"
+            :style="{ width: width, height: height }"
+            class="border border-light rounded-circle"
+          >
+            <pattern
+              id="checkerboard"
+              width="12"
+              height="12"
+              patternUnits="userSpaceOnUse"
+              fill="FFF"
+            >
+              <rect
+                fill="#7080707f"
+                x="0"
+                width="6"
+                height="6"
+                y="0"
+              />
+              <rect
+                fill="#7080707f"
+                x="6"
+                width="6"
+                height="6"
+                y="6"
+              />
+            </pattern>
+
+            <circle
+              cx="16"
+              cy="16"
+              r="16"
+              fill="url(#checkerboard)"
+            />
+
+            <circle
+              cx="16"
+              cy="16"
+              r="16"
+            />
+          </svg>
+        </b-button>
+        <span v-show="showColorCodeText" class="ml-2">
```

Comment:
I would do a v-if, also the text should look like a normal span, it looks dark here